src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -118,6 +118,13 @@ public function dump(Data $data)
             $name = substr($name, strrpos($name, '/') + 1);
         }
 
+        if ('cli-server' === PHP_SAPI) {
+            $cloner = new VarCloner();
+            $dumper = new CliDumper('php://stderr', $this->charset);
+            $dumper->dump($cloner->cloneVar(sprintf('In %s on line %d', $file, $line)));
```

Comment:
shouldn't you just write to ``php://stderr`` instead of using the dumper ? Having all the info about the string does not make much sense IMO, and it could even get truncated by the dumper